README_Docker.md

```diff
@@ -1,114 +1,167 @@
-where DB_PORT is the database port that you choose for this mongodb instance( default being 27017) and mongodb.$DB_PORT is the database directory path for this instance
-While providing the dbpath is purely optional, it gives you a way to gracefully shutdown the mongodb instance using this path when you need to end the instances
 
 Building and running the node container (example, cluster)
----------------------------------------------------------
-#Build a node image
-$docker build -t inode -f Dockerfile-node-cluster .
-(If using proxy, please use
-$docker build -t inode -f Dockerfile-node-cluster --build-arg http_proxy=http://proxy_addr:port . )
+--------------------------------------
+1. Build a node image
+```
+$ docker build -t inode -f Dockerfile-node-cluster .
+```
+_If using proxy, please add a build-arg for the http_proxy: `$ docker build -t inode -f Dockerfile-node-cluster --build-arg http_proxy=http://proxy_addr:port .`_
+
+2. Run the node container
+```
+$ docker run -itd -p 9000:9000 --net node-dc-net -e CPU_COUNT=0 -e DB_URL=mongodb://cmongo:27017/node-dc-eis-db-9000 --name cnodemongo-9000 inode
 
-$docker run -itd -p $NODE_SERVER_PORT:9000 --net node-dc-net -e CPU_COUNT=${cpu_count} -e DB_URL=mongodb://${db_ip}:$DB_PORT/node-dc-eis-db-$NODE_SERVER_PORT --name cnodemongo-$NODE_SERVER_PORT inode
-where NODE_SERVER_PORT and DB_PORT are any available ports for the node server and mongo database that you choose to publish respectively. Be sure to set these values in your environment in your preferred way(a script or export commands)
-Note the environment variables, CPU_COUNT and DB_URL passed as parameters here, and you are free to configure it to the mode you want. The default CPU_COUNT is set to 0 for monolithic mode, and DB_URL is set to the localhost database in Node-DC-EIS-cluster/config/configuration.js
--p NODE_SERVER_PORT:9000 here maps the NODE_SERVER_PORT of your container to 9000 on localhost making the container accessible to host for communication
+```
 
 Alternately, you could also pin the containers to specific cores you want. As an example,
-$docker run -itd --cpuset-cpus 0,1,2,3 -p $NODE_SERVER_PORT:9000 --net node-dc-net -e DB_URL=mongodb://${db_ip}:$DB_PORT/node-dc-eis-db-$NODE_SERVER_PORT --name cnodemongo-$NODE_SERVER_PORT inode
 
-#Execute the node-cluster container bash
-$docker exec -it cnode bash
-You're now inside the node server container
-For simple sanity checks, do a "ps -aux | grep node" and see that the number of node servers are indeed the same as you set
-$exit
+```
+$ docker run -itd --cpuset-cpus 0,1,2,3 -p 9000:9000 --net node-dc-net -e DB_URL=mongodb://cmongo:27017/node-dc-eis-db-9000 --name cnodemongo-9000 inode
+```
 
-#Check for your node-database connection
-It's a good idea to ensure that you're able to reach the database through your container
-$curl --noproxy ${db_ip} --silent http://${db_ip}:$NODE_SERVER_PORT/
-Should return OK
```

Comment:
@boazsender In this scenario the DB server is running outside the container at some db_ip:db_port endpoint. We would like to keep and show that flexibility instead of using 'cmongo' which confuses the setup little bit.

Reply:
Reversion coming shortly. Also leaving in a reference to the host name as a secondary option. Let me know what you think.